### PR TITLE
Adding policy preview screen

### PR DESCRIPTION
**Pull request by jaind1046**
Adding policy preview screen

### CODE REVIEW

File: src/pages/policy.page.js [hunk 1]
```diff
@@ -246,12 +246,12 @@ module.exports = {
    * ***************************************************************
    */
   clickView() {
-    const element = this.buttons.view;
+    const element = this.table.viewPolicyFirst;
```
Comment (NadHodeGW) [inline src/pages/policy.page.js:249]: The element "view" is a button, please update it back to previous state

File: src/pages/policy.page.js [hunk 2]
```diff
@@ -246,12 +246,12 @@ module.exports = {
...
     I.click(element);
   },
 
   clickActivate() {
-    const element = this.buttons.activate;
+    const element = this.table.activatePolicyFirst;
```
Comment (NadHodeGW) [inline src/pages/policy.page.js:254]: The element "activate" is a button, please update it back to previous state

File: specs/features/content-management-previous-policy.feature [hunk 3]
```diff
@@ -2,8 +2,8 @@ Feature: content-management-previous-policy
     As a admin I need to view, validate the accuracy of and activate previous policy details in order to confirm that the solution works as expected
 
     Background:
-        Given I am logged into the ui
-        Given I have navigated to the Policy page
+        Given I am logged into the portal
+        Given I am on current policy screen
 
     @TEST-154
```
Comment (NadHodeGW) [inline specs/features/content-management-previous-policy.feature:8]: please remove this scenario 154, not required

File: specs/features/content-management-previous-policy.feature [hunk 4]
```diff
@@ -13,12 +13,12 @@ Feature: content-management-previous-policy
 
     @TEST-156
     Scenario: I can view previous Policy details
-        Given I have navigated to the Previous Policy tab
+        Given I click on Policy History in the navigation panel
```
Comment (NadHodeGW) [inline specs/features/content-management-previous-policy.feature:16]: this should be changed to: 'Given I am on the Policy History page' to be in line with the other feature files

**Review by NadHodeGW — CHANGES_REQUESTED** [review]
See comments

File: src/pages/policy.page.js (hunk 1 above)
Comment (jaind1046) [reply src/pages/policy.page.js]: Agree. Done.

File: specs/features/content-management-previous-policy.feature (hunk 3 above)
Comment (jaind1046) [reply specs/features/content-management-previous-policy.feature]: Done. tnanks.

File: src/pages/policy.page.js (hunk 2 above)
Comment (jaind1046) [reply src/pages/policy.page.js]: Done thanks.

**Comment by jaind1046** [comment]
@NadHodeGW Incorporated the comments. Thanks.

**Review by NadHodeGW — APPROVED** [review]

File: specs/features/content-management-previous-policy.feature (hunk 4 above)
Comment (jaind1046) [reply specs/features/content-management-previous-policy.feature]: Done thanks.

**Pull request merged**